src/objecttypes/setup_configuration/tests/test_token_auth_config.py

```diff
@@ -0,0 +1,441 @@
+from io import StringIO
+from pathlib import Path
+
+from django.core.management import CommandError, call_command
+from django.test import TestCase
+
+from django_setup_configuration.exceptions import (
+    ConfigurationRunFailed,
+    PrerequisiteFailed,
+)
+from django_setup_configuration.test_utils import build_step_config_from_sources
+
+from objecttypes.setup_configuration.steps import TokenAuthConfigurationStep
+from objecttypes.token.models import TokenAuth
+from objecttypes.token.tests.factories.token import TokenAuthFactory
+
+DIR_FILES = (Path(__file__).parent / "files/token_auth").resolve()
+
+
+class TokenAuthConfigurationStepTests(TestCase):
```

Comment:
Can you also add a test to verify that the step is idempotent (running it twice should not add extra tokens)

See also: https://github.com/maykinmedia/open-klant/pull/297/files#diff-3a5d5be8dde9a748646e2899914d3ff70e49d6041469c440435c436142f95c22R238

Reply:
Done